lib/ptz/ptz.ex

```diff
@@ -14,7 +14,7 @@ defmodule Onvif.PTZ do
   ]
 
   @spec request(Device.t(), module()) :: {:ok, any} | {:error, map()}
-  @spec request(Device.t(), list(), atom()) :: {:ok, any} | {:error, map()}
+  @spec request(Device.t(), any(), atom()) :: {:ok, any} | {:error, map()}
```

Comment:
I think we may have diverged contracts at some point, where the API modules for Media do MFA apply with the args for flexibility but the other API modules are just passing them through. Any opinion on unifying these contracts? I think with the more complex Media calls we end up using a struct for the request args anyway, so passing through single args in the API contract may make more sense.